pkg/users/account-details.js

```diff
@@ -228,6 +224,20 @@
     else
         last_login = timeformat.dateTime(new Date(account.lastLogin));
 
+    const actions = superuser.allowed && (
+        <>
+            <Button variant="secondary" onClick={() => logout_account()} id="account-logout"
+              isDisabled={!account.loggedIn || account.uid == 0}>
+                {_("Terminate session")}
+            </Button>
+            { "\n" }
+            <Button isDisabled={account.uid == 0} variant="danger" id="account-delete"
+                  onClick={() => delete_account_dialog(account)}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18721-20230426-094458-1180deae-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)